lisa/features/nvme.py

```diff
@@ -85,6 +96,18 @@ def get_devices_from_lspci(self) -> List[PciDevice]:
     def get_raw_data_disks(self) -> List[str]:
         return self.get_namespaces()
 
+    def get_raw_nvme_disks(self) -> List[str]:
+        nvme_namespaces = self.get_namespaces()
+        # With disk controller type NVMe, OS disk and all remote disks appear as NVMes.
+        # They should be removed from the list of namespaces for NVMe tests as they are
+        # not actual NVMe devices.
+        # OS disk and all remote disks are connected to the same NVMe controller.
+        # Excluding the OS disk's namespace from the list of namespaces will exclude
+        # all remote disks.
+        get_os_partition_namespace = self.get_os_partition_namespace()
```

Comment:
The variable name should be like `os_partition_namespace` without get.

Reply:
Fixed